lib/ecto/adapters/sql.ex

```diff
@@ -993,6 +998,17 @@ defmodule Ecto.Adapters.SQL do
     end
   end
 
+  @doc false
+  def quote_name(name) when is_atom(name) do
+    quote_name(Atom.to_string(name))
+  end
+  def quote_name(name) do
+    if String.contains?(name, "\"") do
+      raise "bad field name to quote #{inspect name}"
+    end
+    [?<, name, ?>]
```

Comment:
Why is the default `<` and `>`? Versus using `"`. If the default was to use double quotes, I wouldn't need to implement this for the sqlite adapter, and the postgres adapter wouldn't need to overload.

Reply:
I put this here as a reminder that its a default implementation.  Postgres, MyXQL and TDS are all different delimiters and I wasn't sure therefore what a sensible default it. More than happy to change this to `"` if that's considered the same default.